src/h3/_cy/latlng.pyx

```diff
@@ -221,8 +221,8 @@ def polygon_to_cells_experimental(outer, int res, int flags, holes=None):
         A ring given by a sequence of lat/lng pairs.
     res : int
         The resolution of the output hexagons
-    flags : int
-        Polygon to cells flags, such as containment mode.
+    flag : int
```

Comment:
This does not seem like an idiomatic change to me. While containment modes cannot be combined, I think the intention was to expand flags to cover other cases such as spherical vs Cartesian.

Reply:
The Cython function are internal and not part of the externally supported API, so we're free to change this in the future without officially breaking anything.